### PR TITLE
feat(rust): rename ockam forwarder commands to ockam relay

### LINKED ISSUE #4609: rename `ockam forwarder` clap commands to `ockam relay`
### Current behavior

[Currently there are several `ockam forwarder` commands](https://github.com/build-trust/ockam/blob/develop/implementations/rust/ockam/ockam_command/src/forwarder/mod.rs)

```sh
ockam forwarder create
ockam forwarder list
ockam forwader show
```

As well as documentation, code examples, and bats integration tests that refer to these commands.

### Desired behavior

Update the naming of these commands to `ockam relay`

```sh
ockam relay create
ockam relay list
ockam relay show
```
and surrounding documentation, code examples, and bats integration test

---

_We love helping new contributors!_
_If you have questions or need help as you work on your first Ockam contribution, please leave a comment on [this discussion](https://github.com/ockam-network/ockam/discussions/1081)._
_If you're looking for other issues to contribute to, checkout [this discussion](https://github.com/build-trust/ockam/discussions/3206) and labels -  https://github.com/build-trust/ockam/labels/good%20first%20issue or https://github.com/build-trust/ockam/labels/help%20wanted_

### PR DESCRIPTION
**Pull request by caoakleyii**
## Current behavior

```sh
ockam forwarder create
ockam forwarder list
ockam forwarder show
```

## Proposed changes

```sh
ockam relay create
ockam relay list
ockam relay show
```

Resolves https://github.com/build-trust/ockam/issues/4609

## Checks

- [x] All commits in this Pull Request are [signed](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) and Verified by Github.
- [x] All commits in this Pull Request follow the Ockam [commit message convention](https://github.com/build-trust/.github/blob/main/CONTRIBUTING.md#commit-messages).
- [x] I accept the Ockam Community [Code of Conduct](https://github.com/build-trust/.github/blob/main/CODE_OF_CONDUCT.md).
- [x] I have accepted the Ockam [Contributor License Agreement](https://github.com/build-trust/ockam-contributors/blob/main/CLA.md) by adding my Git/Github details in a row at the end of the [CONTRIBUTORS.csv](https://github.com/build-trust/ockam-contributors/blob/main/CONTRIBUTORS.csv) file in a separate pull request to the [build-trust/ockam-contributors](https://github.com/build-trust/ockam-contributors) repository. The easiest way to do this is to [edit the CONTRIBUTORS.csv](https://github.com/build-trust/ockam-contributors/edit/main/CONTRIBUTORS.csv) file in the github web UI and create a PR, this will mark the commit as verified.

### CODE REVIEW

File: implementations/rust/ockam/ockam_command/src/relay/show.rs [hunk 1]
```diff
@@ -33,23 +33,14 @@ async fn run_impl(ctx: Context, (opts, cmd): (CommandGlobalOpts, ShowCommand)) -
     let mut rpc = Rpc::background(&ctx, &opts, &node_name)?;
     rpc.request(Request::get(format!("/node/forwarder/{remote_address}")))
         .await?;
-    let forwarder_info_response = rpc.parse_response::<ForwarderInfo>()?;
+    let relay_info_response = rpc.parse_response::<ForwarderInfo>()?;
```
Comment (etorreborre) [inline implementations/rust/ockam/ockam_command/src/relay/show.rs:36]: I think we should go all the way down and also rename `ForwarderInfo`, `ockam::Forwarder` etc... in order to keep our vocabulary consistent across the whole stack

Comment (caoakleyii) [reply implementations/rust/ockam/ockam_command/src/relay/show.rs:36]: I agree. I had two thoughts though.

 I didn't want to balloon this PR; so I just started with this one crate, `ockam_command`
 
 Also, wanted to be sure it was applicable moving all the way down the stack.

**Comment by adrianbenavides** [comment]
Before merging this PR, we should have ready a PR at https://github.com/build-trust/ockam-documentation fixing the public documentation examples.

**Comment by caoakleyii** [comment]
> Before merging this PR, we should have ready a PR at https://github.com/build-trust/ockam-documentation fixing the public documentation examples.

https://github.com/build-trust/ockam-documentation/pull/10

**Review by mrinalwadhwa — APPROVED** [review]

**Pull request merged**